File: php-pecl/send.php
<?php
$connection = new AMQPConnection();
$connection->connect();

$exchange = new AMQPExchange($connection, '');

$queue = new AMQPQueue($connection);
$queue->declare('hello');

$exchange->publish('Hello World!', 'hello');

echo " [x] Sent 'Hello World!'\n";

$connection->disconnect();

File: clojure/src/rabbitmq/tutorials/send.clj
(ns rabbitmq.tutorials.send
  (:require [langohr.core :as lc]
            [langohr.channel :as lch]
            [langohr.queue :as lq]
            [langohr.basic :as lb]))


(defn -main
  [& args]
  (with-open [conn (lc/connect)]
    (let [ch   (lch/open conn)]
      (lq/declare ch "hello" {:durable false :auto-delete false})
      (lb/publish ch "" "hello" (.getBytes "Hello World!" "UTF-8"))
      (println " [x] Sent 'Hello World!'"))))
